WORKSPACE

```diff
@@ -676,3 +676,31 @@ http_archive(
     integrity = "sha256-H2yX4kuLyNdBrkRPhTr61FQqJRyiKeLq4TnMmKE0t2A=",
     url = "https://github.com/astral-sh/ruff/releases/download/0.6.7/ruff-x86_64-pc-windows-msvc.zip",
 )
+
+http_file(
+    name = "clang-format-darwin-arm64",
+    executable = True,
+    integrity = "sha256-1hG7AcfgGL+IBrSCEhD9ed6pvIpZMdXMdhCDGkqzhpA=",
+    url = "https://github.com/npaun/bins/releases/download/llvm-18.1.8/llvm-18.1.8-darwin-arm64-clang-format",
+)
+
+http_file(
+    name = "clang-format-linux-arm64",
+    executable = True,
+    integrity = "sha256-iCbaPg60x60eA9ZIWmSdFva/RD9xOBcJLUwSRK8Gxzk=",
+    url = "https://github.com/npaun/bins/releases/download/llvm-18.1.8/llvm-18.1.8-linux-arm64-clang-format",
```

Comment:
Can we move this to a repository inside Cloudflare organization?

Reply:
Soon hopefully

Reply:
This can be merged and we can move it to a CF org repo later, it'll involve some tickets and stuff and shouldn't block enabling formatting again.

Reply:
A comment in here explaining that would be helpful